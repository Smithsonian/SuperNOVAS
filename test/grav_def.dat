
2450545.000000 TEST       S2 O0 A1: 33571570.267636 -19382561.638421 14109315.573232 

2450545.000000 TEST       S2 O1 A1: 33571563.176026 -19382553.194655 14109344.046431 

2450545.000000 TEST       S2 O2 A1: 33571564.215470 -19382551.633000 14109343.718501 

2450545.000000 SUN        S0 O0 A1:    -0.006750     0.004995     0.002339 

2450545.000000 SUN        S0 O1 A1:    -0.006750     0.004995     0.002339 

2450545.000000 SUN        S0 O2 A1:    -0.006750     0.004995     0.002339 

2451545.000000 TEST       S2 O0 A1: 33571591.004648 -19382509.842733 14109337.385632 

2451545.000000 TEST       S2 O1 A1: 33571564.453030 -19382550.637533 14109344.520764 

2451545.000000 TEST       S2 O2 A1: 33571564.215459 -19382551.632996 14109343.718531 

2451545.000000 SUN        S0 O0 A1:    -0.007138    -0.002644    -0.000921 

2451545.000000 SUN        S0 O1 A1:    -0.007138    -0.002644    -0.000921 

2451545.000000 SUN        S0 O2 A1:    -0.007138    -0.002644    -0.000921 

2452545.000000 TEST       S2 O0 A1: 33571615.943505 -19382361.436194 14109481.918292 

2452545.000000 TEST       S2 O1 A1: 33571564.305025 -19382551.473800 14109343.724111 

2452545.000000 TEST       S2 O2 A1: 33571564.215430 -19382551.633062 14109343.718509 

2452545.000000 SUN        S0 O0 A1:     0.000657    -0.004704    -0.002013 

2452545.000000 SUN        S0 O1 A1:     0.000657    -0.004704    -0.002013 

2452545.000000 SUN        S0 O2 A1:     0.000657    -0.004704    -0.002013 
