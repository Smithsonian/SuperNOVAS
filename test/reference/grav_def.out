
-10000.0 22+20      S2 O0 A0: ERROR 13 ERROR 13 

-10000.0 22+20      S2 O1 A0: ERROR 13 ERROR 13 

-10000.0 22+20      S2 O2 A0: ERROR 13 ERROR 13 

-10000.0 16-20      S2 O0 A0: ERROR 13 ERROR 13 

-10000.0 16-20      S2 O1 A0: ERROR 13 ERROR 13 

-10000.0 16-20      S2 O2 A0: ERROR 13 ERROR 13 

-10000.0 SUN        S0 O0 A0: ERROR 13 

-10000.0 SUN        S0 O1 A0: ERROR 13 

-10000.0 SUN        S0 O2 A0: ERROR 13 

-10000.0 EARTH      S0 O0 A0: ERROR 13 ERROR 13 

-10000.0 EARTH      S0 O1 A0: ERROR 13 ERROR 13 

-10000.0 EARTH      S0 O2 A0: ERROR 13 ERROR 13 

     0.0 22+20      S2 O0 A0: ERROR 13 ERROR 13 

     0.0 22+20      S2 O1 A0: ERROR 13 ERROR 13 

     0.0 22+20      S2 O2 A0: ERROR 13 ERROR 13 

     0.0 16-20      S2 O0 A0: ERROR 13 ERROR 13 

     0.0 16-20      S2 O1 A0: ERROR 13 ERROR 13 

     0.0 16-20      S2 O2 A0: ERROR 13 ERROR 13 

     0.0 SUN        S0 O0 A0: ERROR 13 

     0.0 SUN        S0 O1 A0: ERROR 13 

     0.0 SUN        S0 O2 A0: ERROR 13 

     0.0 EARTH      S0 O0 A0: ERROR 13 ERROR 13 

     0.0 EARTH      S0 O1 A0: ERROR 13 ERROR 13 

     0.0 EARTH      S0 O2 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 SUN        S0 O0 A0: ERROR 13 

 10000.0 SUN        S0 O1 A0: ERROR 13 

 10000.0 SUN        S0 O2 A0: ERROR 13 

 10000.0 EARTH      S0 O0 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O1 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O2 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 SUN        S0 O0 A0: ERROR 13 

 10000.0 SUN        S0 O1 A0: ERROR 13 

 10000.0 SUN        S0 O2 A0: ERROR 13 

 10000.0 EARTH      S0 O0 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O1 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O2 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 22+20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O0 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O1 A0: ERROR 13 ERROR 13 

 10000.0 16-20      S2 O2 A0: ERROR 13 ERROR 13 

 10000.0 SUN        S0 O0 A0: ERROR 13 

 10000.0 SUN        S0 O1 A0: ERROR 13 

 10000.0 SUN        S0 O2 A0: ERROR 13 

 10000.0 EARTH      S0 O0 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O1 A0: ERROR 13 ERROR 13 

 10000.0 EARTH      S0 O2 A0: ERROR 13 ERROR 13 

-10000.0 22+20      S2 O0 A1:   0.8137968  -0.4698492   0.3420183   0.8444860  -0.0961578   0.5268749 

-10000.0 22+20      S2 O1 A1:   0.8137977  -0.4698463   0.3420201   0.8444856  -0.0961545   0.5268761 

-10000.0 22+20      S2 O2 A1: ERROR 11 ERROR 11 

-10000.0 16-20      S2 O0 A1:  -0.4698452  -0.8137982  -0.3420204  -0.7069396  -0.6555005  -0.2656229 

-10000.0 16-20      S2 O1 A1:  -0.4698463  -0.8137977  -0.3420201  -0.7069405  -0.6554997  -0.2656225 

-10000.0 16-20      S2 O2 A1: ERROR 11 ERROR 11 

-10000.0 SUN        S0 O0 A1:  -0.0022285   0.9160895   0.4009677 

-10000.0 SUN        S0 O1 A1:  -0.0022248   0.9160895   0.4009676 

-10000.0 SUN        S0 O2 A1: ERROR 11 

-10000.0 EARTH      S0 O0 A1:   0.8018803  -0.5481741  -0.2376829   0.9741757  -0.2078379  -0.0882329 

-10000.0 EARTH      S0 O1 A1:   0.8018817  -0.5481724  -0.2376822   0.9741763  -0.2078358  -0.0882320 

-10000.0 EARTH      S0 O2 A1: ERROR 11 ERROR 11 

     0.0 22+20      S2 O0 A1:   0.8137983  -0.4698453   0.3420200   0.1279366  -0.9388945   0.3195455 

     0.0 22+20      S2 O1 A1:   0.8137977  -0.4698463   0.3420202   0.1279354  -0.9388947   0.3195453 

     0.0 22+20      S2 O2 A1: ERROR 11 ERROR 11 

     0.0 16-20      S2 O0 A1:  -0.4698402  -0.8138006  -0.3420216  -0.7134918  -0.6499481  -0.2617192 

     0.0 16-20      S2 O1 A1:  -0.4698464  -0.8137977  -0.3420201  -0.7134967  -0.6499436  -0.2617170 

     0.0 16-20      S2 O2 A1: ERROR 11 ERROR 11 

     0.0 SUN        S0 O0 A1:  -0.9309433  -0.3448237  -0.1201711 

     0.0 SUN        S0 O1 A1:  -0.9309441  -0.3448220  -0.1201704 

     0.0 SUN        S0 O2 A1: ERROR 11 

     0.0 EARTH      S0 O0 A1:  -0.1876672   0.9010974   0.3909022  -0.7884052   0.5568965   0.2613110 

     0.0 EARTH      S0 O1 A1:  -0.1876693   0.9010970   0.3909021  -0.7884066   0.5568950   0.2613103 

     0.0 EARTH      S0 O2 A1: ERROR 11 ERROR 11 

 10000.0 22+20      S2 O0 A1:   0.8137993  -0.4698412   0.3420233   0.6747634  -0.7235697   0.1454005 

 10000.0 22+20      S2 O1 A1:   0.8137977  -0.4698463   0.3420201   0.6747601  -0.7235735   0.1453968 

 10000.0 22+20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 16-20      S2 O0 A1:  -0.4698624  -0.8137897  -0.3420170  -0.3364820  -0.8686379  -0.3636593 

 10000.0 16-20      S2 O1 A1:  -0.4698463  -0.8137977  -0.3420201  -0.3364648  -0.8686436  -0.3636615 

 10000.0 16-20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 SUN        S0 O0 A1:  -0.0022809  -0.9234995  -0.3835928 

 10000.0 SUN        S0 O1 A1:  -0.0022829  -0.9234995  -0.3835928 

 10000.0 SUN        S0 O2 A1: ERROR 11 

 10000.0 EARTH      S0 O0 A1:  -0.5291448  -0.7785606  -0.3374153  -0.3817534  -0.8499384  -0.3631377 

 10000.0 EARTH      S0 O1 A1:  -0.5291314  -0.7785685  -0.3374182  -0.3817387  -0.8499442  -0.3631397 

 10000.0 EARTH      S0 O2 A1: ERROR 11 ERROR 11 

 10000.0 22+20      S2 O0 A1:   0.8137993  -0.4698412   0.3420233   0.6747634  -0.7235697   0.1454005 

 10000.0 22+20      S2 O1 A1:   0.8137977  -0.4698463   0.3420201   0.6747601  -0.7235735   0.1453968 

 10000.0 22+20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 16-20      S2 O0 A1:  -0.4698624  -0.8137897  -0.3420170  -0.3364820  -0.8686379  -0.3636593 

 10000.0 16-20      S2 O1 A1:  -0.4698463  -0.8137977  -0.3420201  -0.3364648  -0.8686436  -0.3636615 

 10000.0 16-20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 SUN        S0 O0 A1:  -0.0022809  -0.9234995  -0.3835928 

 10000.0 SUN        S0 O1 A1:  -0.0022829  -0.9234995  -0.3835928 

 10000.0 SUN        S0 O2 A1: ERROR 11 

 10000.0 EARTH      S0 O0 A1:  -0.5291448  -0.7785606  -0.3374153  -0.3817534  -0.8499384  -0.3631377 

 10000.0 EARTH      S0 O1 A1:  -0.5291314  -0.7785685  -0.3374182  -0.3817387  -0.8499442  -0.3631397 

 10000.0 EARTH      S0 O2 A1: ERROR 11 ERROR 11 

 10000.0 22+20      S2 O0 A1:   0.8137993  -0.4698412   0.3420233   0.6747668  -0.7235662   0.1454017 

 10000.0 22+20      S2 O1 A1:   0.8137977  -0.4698463   0.3420201   0.6747636  -0.7235700   0.1453980 

 10000.0 22+20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 16-20      S2 O0 A1:  -0.4698624  -0.8137897  -0.3420170  -0.3364801  -0.8686385  -0.3636596 

 10000.0 16-20      S2 O1 A1:  -0.4698463  -0.8137977  -0.3420201  -0.3364629  -0.8686442  -0.3636618 

 10000.0 16-20      S2 O2 A1: ERROR 11 ERROR 11 

 10000.0 SUN        S0 O0 A1:  -0.0022705  -0.9234995  -0.3835929 

 10000.0 SUN        S0 O1 A1:  -0.0022725  -0.9234995  -0.3835928 

 10000.0 SUN        S0 O2 A1: ERROR 11 

 10000.0 EARTH      S0 O0 A1:  -0.5290028  -0.7786419  -0.3374505  -0.3816418  -0.8499807  -0.3631561 

 10000.0 EARTH      S0 O1 A1:  -0.5289893  -0.7786498  -0.3374534  -0.3816271  -0.8499864  -0.3631581 

 10000.0 EARTH      S0 O2 A1: ERROR 11 ERROR 11 
